### DISCUSSION TITLE
vita2hos

### DISCUSSION THREAD

**Discussion by thoriqbayu** [skyline-emu/skyline#533] [Game Requests]
I think it will run natively even on emulator, so please add support for this amazing homebrew! thank you dev

**Comment by PixelyIon** [comment]
Interesting request, I've actually helped out @xerpi a bit with vita2hos and we did talk about this. It will likely take a good while till we run 32-bit homebrew/games, it's definitely not going to be this year. It is something we do want to support in the long run given that Vita3k doesn't do an Android port themselves.
